numpy/_core/src/multiarray/unique.h

```diff
@@ -0,0 +1,16 @@
+#ifndef NUMPY_CORE_INCLUDE_NUMPY_MULTIARRAY_UNIQUE_H_
+#define NUMPY_CORE_INCLUDE_NUMPY_MULTIARRAY_UNIQUE_H_
+
+#include <Python.h>
```

Comment:
I don't think this should be in a header - we keep all includes in the object files